sdk/messaging_servicebus/src/service_bus/mod.rs

```diff
@@ -89,17 +89,25 @@ async fn send_message(
     policy_name: &str,
     signing_key: &Secret,
     msg: &str,
+    broker_properties: Option<SettableBrokerProperties>,
```

Comment:
I'd be fine with the breaking change, but if we're going to break it once - since it's all unsupported and prerelease anyway - lets keep it at once. Instead, it would be better to have an options bag following our current guidelines in part, using the method name as the root + "Options" e.g., `SendMessageOptions` or, rather, `Option<SendMessageOptions>`. That should have member(s) including this `broker_properties` header value. It might also be worth it to implement `AsHeader` like all our other headers in `azure_core`, even if only for this crate since it's special-purposed to Service Bus.

Reply:
Thanks for the feedback !
I just updated the PR, I also added all the other options in BrokerProperties (I did not test all of them)

I just not sure about the `unwrap` in the `AsHeaders` implementation